ide/editor.settings.storage/src/org/netbeans/modules/editor/settings/storage/fontscolors/ColoringStorage.java

```diff
@@ -305,8 +311,8 @@ public void run() throws IOException {
     private static final String V_ITALIC = "italic"; //NOI18N
     private static final String V_PLAIN = "plain"; //NOI18N
     
-    private static final String PUBLIC_ID = "-//NetBeans//DTD Editor Fonts and Colors settings 1.1//EN"; //NOI18N
-    private static final String SYSTEM_ID = "http://www.netbeans.org/dtds/EditorFontsColors-1_1.dtd"; //NOI18N
+    private static final String PUBLIC_ID = "-//NetBeans//DTD Editor Fonts and Colors settings 1.2//EN"; //NOI18N
+    private static final String SYSTEM_ID = "http://netbeans.apache.org/dtds/EditorFontsColors-1_2.dtd"; //NOI18N
```

Comment:
Can we still correctly read settings of the previous format?

Reply:
Yes. Tests can approve that.